core/scripts/sql/update/14.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS cluster (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    owner_id INT UNSIGNED NOT NULL,
+    machine_type VARCHAR(255) NOT NULL,
+    number_of_machines INT NOT NULL,
+    creation_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    status ENUM('LAUNCHING', 'LAUNCHED', 'PAUSING', 'PAUSED', 'RESUMING', 'TERMINATING', 'TERMINATED'),
+    FOREIGN KEY (owner_id) REFERENCES user (uid) ON DELETE CASCADE
+) ENGINE = INNODB;
+
+CREATE TABLE IF NOT EXISTS cluster_activity (
+    `id` INT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Is this table necessary?

Reply:
Yes. We need this table to keep track of the history of the running time of the cluster.